examples/operator-rbac-openshift-tls/1-setup-operator-rbac.ipynb

```diff
@@ -0,0 +1,741 @@
+    "## Objective\n",
+    "\n",
+    "This demo provides a reference implementation of a runbook on how to enable Role-Based Access Control (RBAC) for Feast using the Feast Operator with the Kubernetes authentication type. This serves as useful reference material for a cluster admin / MLOps engineer.\n",
+    "\n",
+    "The demo steps include deploying the Feast Operator, creating Feast instances with server components (registry, offline store, online store), and Feast client testing locally and from the Openshift. The goal is to ensure secure access control for Feast instances deployed by the Feast Operator.\n",
+    " \n",
+    "Please read these reference documents for understanding the Feast RBAC framework.\n",
+    "- [RBAC Architecture](https://docs.feast.dev/v/master/getting-started/architecture/rbac) \n",
+    "- [RBAC Permission](https://docs.feast.dev/v/master/getting-started/concepts/permission).\n",
+    "- [RBAC Authorization Manager](https://docs.feast.dev/v/master/getting-started/components/authz_manager)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Deployment Architecture\n",
+    "In this notebook, we will deploy a distributed topology of Feast services, which includes:\n",
+    "\n",
+    "* `Registry Server`: Handles metadata storage for feature definitions.\n",
+    "* `Online Store Server`: Uses the `Registry Server` to query metadata and is responsible for low-latency serving of features.\n",
+    "* `Offline Store Server`: Uses the `Registry Server` to query metadata and provides access to batch data for historical feature retrieval.\n",
+    "\n",
+    "Additionally, we will cover:\n",
+    "* RBAC Configuration with Kubernetes Authentication for Feast resources."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Prerequisites\n",
+    "* Openshift Cluster\n",
+    "* [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl) Kubernetes CLI tool.\n",
+    "* Authenticate to an existing OpenShift cluster."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2025-03-14T14:47:46.558319Z",
+     "start_time": "2025-03-14T14:47:46.117084Z"
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "namespace/feast created\n",
+      "Context \"default/api-jyejare-cluster-h9lx-p3-openshiftapps-com:443/jyejare\" modified.\n"
+     ]
+    }
+   ],
+   "source": [
+    "!kubectl create ns feast\n",
+    "!kubectl config set-context --current --namespace feast"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Validate the cluster setup:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2025-03-14T14:47:52.996466Z",
+     "start_time": "2025-03-14T14:47:52.827264Z"
```

Comment:
please run every changed notebook against live clusters and commit the outputs

Reply:
I did run the same, just that the timestamps are not changing.